style/scripts/taginfo_template.json

```diff
@@ -0,0 +1,19 @@
+{
+  "data_format": 1,
+  "project": {
+    "name": "OpenStreetMap Americana",
+    "description": "A quintessentially American map style.",
+    "project_url": "https://github.com/ZeLonewolf/openstreetmap-americana/",
+    "doc_url": "https://wiki.openstreetmap.org/wiki/OpenStreetMap_Americana",
+    "icon_url": "https://raw.githubusercontent.com/ZeLonewolf/openstreetmap-americana/main/doc-img/osm-americana-logo.png",
```

Comment:
The taginfo projects page [says](https://wiki.openstreetmap.org/wiki/Taginfo/Projects):
> project logo, should work in 16x16 pixels on white and light gray backgrounds

Do we need to provide a 16x16 version or does taginfo shrink the image down to size?  We do also have a favico.ico that can potentially be used.

Reply:
It seems to generate its own copy of the image at 16×16. I’m not sure if it supports .ico format. It only seems to copy the project image, not the individual tag icons. It embeds those directly, using CSS to control the size.